k4MarlinWrapper/examples/event_display.py

```diff
@@ -56,6 +58,118 @@
 io_handler = IOHandlerHelper(algList, iosvc)
 io_handler.add_reader(reco_args.inputFiles)
 
+ColConfig = namedtuple("ColConfig", ["col_name", "val1", "val2", "col_toggle_group"])
```

Comment:
```suggestion
ColDrawConfig = namedtuple("ColConfig", ["col_name", "marker_type", "size", "toggle_group"])
```

`marker_type` and `size` from [here](https://github.com/iLCSoft/CEDViewer/blob/50c041343a4b069f7a7458e7d98267f349465bc2/src/DDCEDViewer.cc#L71-L75)